connectors/rsk.go

```diff
@@ -103,6 +105,7 @@ func (rsk *RSK) Connect(endpoint string) error {
 	if err != nil {
 		return err
 	}
+	rsk.lbcCaller = &bindings.LBCCallerRaw{Contract: &rsk.lbc.LBCCaller}
```

Comment:
Why not create this when you need it instead of storing it as a class variable?

Reply:
Eitherway is fine as it's just a pointer to the contract. I did it originally this way cuz I thought it could be easier to access this in case of need. Moved it to a local variable.